packages/hs-react-ui/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -72,12 +73,16 @@ const teaOptions = [
   },
 ];
 
+const StyledPlaceholder = styled(Dropdown.Placeholder)`
+  color: ${colors.primary};
+`;
+
```

Comment:
I vote we don't have a fancy placeholder for the storybook at all - it's hard to know from looking at the component what is/isn't default otherwise
```suggestion
```

Reply:
Sounds good to me.